assert/assertions_test.go

```diff
@@ -2189,3 +2190,22 @@ func Test_validateEqualArgs(t *testing.T) {
 		t.Error("nil functions are equal")
 	}
 }
+
+func Test_truncatingFormat(t *testing.T) {
+
+    original := strings.Repeat("a", bufio.MaxScanTokenSize-102)
+    result := truncatingFormat(original)
+    if fmt.Sprintf("%#v", original) != result {
+        t.Error("string should not be truncated")
+    }
+
+    original = original +"x"
+    result = truncatingFormat(original)
+    if fmt.Sprintf("%#v", original) == result {
```

Comment:
Could you please use the `Equal()` function used throughout the rest of the test file? We prefer to dogfood our own testing framework :)